composer.json

```diff
@@ -69,10 +72,13 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "9.0.x-dev"
+            "dev-main": "4.0.x-dev"
         }
     },
-    "bin": ["bin/ezbehat", "bin/ezreport"],
+    "bin": [
+        "bin/ezbehat",
```

Comment:
I will rebrand this scripts to `ibexabehat` and `ibexareport`, but I want to do it from 8.3 as a follow-up - so that we don't need to maintain different CI scripts that run Behat